workshops/operations.html

```diff
@@ -206,10 +206,8 @@ <h3 id="cost">What About Cost?</h3>
   We charge an administration fee to cover costs such as instructor training,
   curriculum development,
   and so on.
-  As of December 2014,
-  this is US$1250 for the first workshop at any site in a year,
-  and US$750 for each subsequent workshop at that site in the same year,
-  but <a href="{{page.root}}/blog/2015/07/changes-to-admin-fee.html">a new fee schedule is being introduced on September 1, 2015</a>.
+  For the first workshop in a given year at any site, the fee is  US$2,500
+  and US$1,250 for each subsequent workshop at that same site in the same year.
```

Comment:
+1 to merge, or do you want to replace with a link to the SCF page and "check here for latest" wording so that this doesn't fall out of date again?